### PR TITLE
🌐 Add French translation for `docs/fr/docs/help-fastapi.md`

### PR DESCRIPTION
**Pull request by JulianMaurin**
Dear fellow translators 👋

This is a small PR to translate `docs/help-fastapi.md` to french 🇫🇷

Thanks 🍦

### CODE REVIEW

File: docs/fr/docs/help-fastapi.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+## Watch le dépôt GitHub
+
+Vous pouvez "watch" FastAPI dans GitHub (en cliquant sur le bouton "watch" en haut à droite) :<a href="https://github.com/tiangolo/fastapi" class="external-link" target="_blank">https://github.com/tiangolo/fastapi</a>. 👀
+
+Si vous sélectionnez "Watching" au lieu de "Releases only", vous recevrez des notifications lorsque quelqu'un crée une nouvelle Issue.
+
+Vous pouvez alors essayer de les aider à résoudre ces problèmes.
+
+## Créer une Issues
```
Comment (rjNemo) [inline docs/fr/docs/help-fastapi.md:69]:
```suggestion
## Créer une Issue
```

File: docs/fr/docs/help-fastapi.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+Vous pouvez <a href="https://github.com/tiangolo/fastapi/issues/new/choose" class="external-link" target="_blank">créer une Issue</a> dans le dépôt GitHub, par exemple pour :
+
+* Poser une question ou s'informer sur un problème.
+* Suggérer une nouvelle fonctionnalité.
+
+**Note** : si vous créez un problème, alors je vais vous demander d'aider aussi les autres. 😉
+
+## Créer une Pull Request
+
+Vous pouvez <a href="https://github.com/tiangolo/fastapi" class="external-link" target="_blank">créer une Pull Request</a>, par exemple:
```
Comment (rjNemo) [inline docs/fr/docs/help-fastapi.md:80]:
```suggestion
Vous pouvez <a href="https://github.com/tiangolo/fastapi" class="external-link" target="_blank">créer une Pull Request</a>, par exemple :
```

File: docs/fr/docs/help-fastapi.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+Comme vous l'avez vu dans la documentation, FastAPI se tient sur les épaules des géants, Starlette et Pydantic.
+
+Vous pouvez également parrainer :
+
+* <a href="https://github.com/sponsors/samuelcolvin" class="external-link" target="_blank">Samuel Colvin (Pydantic)</a>
+* <a href="https://github.com/sponsors/encode" class="external-link" target="_blank">Encode (Starlette, Uvicorn)</a>
+
+---
+
+Merci! 🚀
```
Comment (rjNemo) [inline docs/fr/docs/help-fastapi.md:122]:
```suggestion
Merci ! 🚀
```

File: docs/fr/docs/help-fastapi.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+Vous pouvez y sélectionner "Releases only".
+
+Ainsi, vous recevrez des notifications (dans votre courrier électronique) chaque fois qu'il y aura une nouvelle version de **FastAPI** avec des corrections de bugs et de nouvelles fonctionnalités.
+
+## Se rapprocher de l'auteur
+
+Vous pouvez vous rapprocher de <a href="https://tiangolo.com" class="external-link" target="_blank">moi (Sebastián Ramírez / `tiangolo`)</a>, l'auteur.
+
+Vous le pouvez :
```
Comment (rjNemo) [inline docs/fr/docs/help-fastapi.md:31]:
```suggestion
Vous pouvez :
```

**Review by rjNemo — APPROVED** [review]

**Review by axel584 — APPROVED** [review]

**Comment by tiangolo** [comment]
Awesome, thanks @JulianMaurin! :cake: 

And thanks for the reviews @axel584 and @rjNemo! :bow:

**Pull request merged**